update-ws-transport/src/main/wsdl/svn.dbc.dk/repos/WSContracts/holdingsItemsUpdate/trunk/holdingsItemsUpdate.xsd

```diff
@@ -132,14 +134,43 @@
       <xs:element name="hasOnlineHolding" type="xs:boolean"/>
     </xs:sequence>
   </xs:complexType>
+  
+  <!-- types -->
+  <xs:complexType name="BorrowerId">
+    <xs:sequence>
+      <xs:element ref="tns:borrowerIdType"/>
+      <xs:element ref="tns:borrowerIdValue"/>
+      <xs:element ref="tns:age" minOccurs="0"/>
+      <xs:element ref="tns:gender" minOccurs="0"/>
+    </xs:sequence>
+  </xs:complexType>
 
+  <xs:element name="borrowerIdType" type="tns:borrowerIdTypes"/>
+  <xs:simpleType name="borrowerIdTypes">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="LOCAL"/>
+      <xs:enumeration value="UNILOGIN"/>
+      <xs:enumeration value="CPR"/>
+      <xs:enumeration value="CVR"/>
```

Comment:
CVR findes ikke i nyeste version af CULR da det ikke er unikt per provider, eller globalt unikt for den sags skyld.